### PR TITLE
Use shared browserslist config

### PR DESCRIPTION
**Pull request by GretaD**
Signed-off-by: greta <gretadoci@gmail.com>

### CODE REVIEW

**Comment by GretaD** [comment]
I dont know how to test this

File: package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
 	},
 	"browserslist": [
 		"last 2 versions",
-		"not ie <= 11"
+		"not ie <= 11",
```
Comment (juliushaertl) [inline package.json:27]: No need for the other two entries anymore if extending from `@nextcloud/browserlist-config` :wink:

**Comment by ChristophWurst** [comment]
> I dont know how to test this

Good question. Possibly took a look at the generated bundles and see if they contain any funky js syntax that our browsers might not understand? It's hard because our new browserslist config only supports modern browsers and they understand most modern js.

**Comment by GretaD** [comment]
> > I dont know how to test this
> 
> Good question. Possibly took a look at the generated bundles and see if they contain any funky js syntax that our browsers might not understand? It's hard because our new browserslist config only supports modern browsers and they understand most modern js.

ok, let me see

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**